src/classes/RundownManager.ts

```diff
@@ -83,8 +83,8 @@ export class RundownManager {
 				if (rawSegment) {
 					const segment: UnrankedSegment = {
 						externalId: rawSegment.identifier,
-						name: rawSegment.fields.title,
-						modified: ParseDateFromInews(rawSegment.fields.modifyDate),
+						name: rawSegment.fields.title || '',
```

Comment:
On this line and the line below I'd prefer the use of `??` over `||`, as it means we'll only use the default in the case of nullish values, not falsy values.